packages/sync/src/Replicastore.php

```diff
@@ -38,6 +38,11 @@ function full_sync_end( $checksum ) {
 		// noop right now
 	}
 
+	public function term_count() {
+		global $wpdb;
+		return $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->terms" );
```

Comment:
I wonder if we could use `wp_count_terms()` with an empty `array()` for the first argument?

Reply:
looks like a `$taxonomy` is required for `wp_count_terms()`

Reply:
The argument is required, but doesn't an empty array satisfy that requirement?

Reply:
Yep, you are right.

Reply:
FWIW I haven't tested with it, so let's make sure it works before using it 😅

Reply:
However, we may need to do a more granular query once we implement the terms blacklist that you are working on. I think i will leave this as is for now, and we can revisit after the blacklist is implemented.

Reply:
Sounds great! Let's add a `TODO` comment here, or open an issue in the 7.6 milestone, so we don't forget about it. After that, feel free to 🚢 . Thanks!

Reply:
Rocco kindly opened #12942, so that's considered ✅ . Let's ship!